core/src/main/java/io/questdb/griffin/SqlCompiler.java

```diff
@@ -1106,6 +1106,7 @@ private CompiledQuery compileUsingModel(SqlExecutionContext executionContext) th
         ExecutionModel executionModel = compileExecutionModel(executionContext);
         switch (executionModel.getModelType()) {
             case ExecutionModel.QUERY:
+                LOG.info().$("Compiled model: ").$((QueryModel) executionModel).$();
```

Comment:
This is going to be hard to trace in multi-user environment. Compiled model for what SQL? Which FD?